### ISSUE TITLE
Focus active manim shell when running "Preview Selected Manim Code"

### ISSUE THREAD

**Issue by bhoov** [Manim-Notebook/manim-notebook#86]
(building a small scene in manim for work and am recording features that would be nice for my workflow)

When debugging a scene, I find that I often want to check the state of data that is underlying the visualization. The nicest way to do these "debug" statements is to type `cmd+' cmd+r` to run selected lines (they may be as simple as printing shapes or values of existing objects). However, I then have to find the terminal that is running the active manim session to see the output.

**Comment by VladimirFokow** [comment]
maybe for this: add a checkbox in the settings?
(if some users want it and some don't)?

**Comment by Splines** [comment]
Ben, would you like that the shell opens _whenever_ you use this shortcut? Or _just_ in some scenarios, e.g. for "debug" statements. E.g. even if we had a checkbox in the settings, it could be beneficial to have a _sperate_ command to run in the shell _AND_  open the respective shell that the command was run in. Since maybe most of the time you don't really need the terminal when just previewing.

Would a command like `Open Active Manim Terminal` be a way to go?

**Comment by bhoov** [comment]
YES. Having an "Open active manim terminal cmd" that can be arbitrarily bound to a hotkey is perfect. Thanks @Splines. 

My idea is that whenever you run a specific selection of lines, you may want to see what happens in the video, or what the data looks like (perhaps w/ print statements) in the terminal. My workflow would say I would always want to see both if I run just specific lines. But let's not develop to my specific workflow right now, this is not an urgent issue
